src/components/search/SearchBox.tsx

```diff
@@ -49,6 +34,33 @@ const SearchBox = (props: SearchBoxProps) => {
             label=""
             placeholder={placeholderText}
             searchFields={searchFields}
+            onSelect={({
+              currentFilter,
+              executeFilterSearch,
+              newDisplayName,
+              newFilter,
+              setCurrentFilter,
+            }) => {
+              // Update static filters.
+              if (currentFilter) {
+                searchActions.setFilterOption({
+                  filter: currentFilter,
+                  selected: false,
+                });
+              }
+              searchActions.setFilterOption({
+                filter: newFilter,
+                displayName: newDisplayName,
+                selected: true,
+              });
+              setCurrentFilter(newFilter);
+              executeFilterSearch(newDisplayName);
```

Comment:
Why do we execute a filtersearch?

Reply:
Doing that to update the dropdown options to use the current query a user supplied after clicking an option. Otherwise the options would be slightly out of date when the user clicks back into the search bar.